website/content/docs/platform/aws/lambda-extension.mdx

```diff
@@ -206,7 +206,7 @@ initialization timeout imposed by the Extensions API when writing files to disk.
 | `VAULT_DEFAULT_CACHE_TTL`     | The time to live configuration (aka, TTL) of the cache used by proxy server. Must have a unit and be parsable as a time.Duration. Required for caching to be enabled.           | No       | `15m`               |
 | `VAULT_DEFAULT_CACHE_ENABLED` | Enable caching for all requests, without needing to set the X-Vault-Cache-Control header for each request. Must be set to a boolean value.                                      | No       | `true`              |
 | `VAULT_ASSUMED_ROLE_ARN`      | Valid ARN of an IAM role that can be assumed by the execution role assigned to your Lambda function.                                                                            | No       | `arn:aws:iam::123456789012:role/xaccounts3access`
-| `VAULT_LOG_LEVEL`             | Log verbosity level, one of TRACE, DEBUG, INFO, WARN, ERROR, OFF. Defaults to INFO.                                                                                             | No       | `DEBUG`
+| `BAO_LOG_LEVEL`               | Log verbosity level, one of TRACE, DEBUG, INFO, WARN, ERROR, OFF. Defaults to INFO.                                                                                             | No       | `DEBUG`
```

Comment:
Technically per above, we didn't fork this and so it would still use the `VAULT_LOG_LEVEL`, but if Jan agrees, we might remove this instead. But I wouldn't hold up merge on this, my 2c.,

Reply:
Oh, this is docs from an extension. I used a command to automatically replace everything at once and then went through each change to understand if they made change, this one might have escaped.

Reply:
So if we don't fork the aws extension, which I agree we shouldn't (at least for now), we keep this documentation file as it was originally?

Reply:
Either we keep it or we remove it entirely. I'd probably just delete it entirely for now, tbh. My 2c.

Reply:
Done :+1: